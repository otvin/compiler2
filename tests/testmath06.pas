program testmath06(output);
begin
	writeln(sqr(1));
	writeln(sqr(0));
	writeln(sqr(-1));
	writeln(sqr(12));
	writeln(sqr(-25));
	writeln(sqr(1234));
	writeln(sqr(-32768));
	writeln(sqr(sqr(2)));
	writeln(sqr(sqr(32768)));
end.
