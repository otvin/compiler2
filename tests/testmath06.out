1
0
1
144
625
1522756
1073741824
16
Overflow error